qemu/tests/numa_stress.py

```diff
@@ -87,6 +87,8 @@ def run(test, params, env):
     most_used_node, memory_used = max_mem_map_node(host_numa_node, qemu_pid)
 
     for test_round in range(test_count):
+        if os.path.isfile(memory_file):
```

Comment:
@yanan-fu  I also hit the issue " Don't have enough memory to execute this test after 1 round" in my host with two numa nodes.  The "os.remove"  fix it  as the memory_file consumed much memory in host.